Makefile

```diff
@@ -196,15 +196,15 @@ $(GIT_UG_REPO_PATH):
 $(GIT_SAMPLE_DATA_REPO_PATH): | $(DATA_DIR)
 	-git clone $(GIT_SAMPLE_DATA_REPO) $(GIT_SAMPLE_DATA_REPO_PATH)
 	git -C $(GIT_SAMPLE_DATA_REPO_PATH) fetch
-	git -C $(GIT_SAMPLE_DATA_REPO_PATH) lfs install
+	-git -C $(GIT_SAMPLE_DATA_REPO_PATH) lfs install
```

Comment:
Well, the `git clone` one, is pretty clear ... if the repo already exists at that path, `git clone` will fail and that's ok.  It's more straightforward to do this than to handle the cross-platform if/else within the recipe.

For `git lfs install`, this one was failing (as far as I could tell) because `git lfs` is already installed and had installed some kind of hook that reinstallation would overwrite. But now that I'm looking for the failing builds, I'm less convinced that skipping on failure here is needed.  Removing this in cd492aa4b